### ISSUE TITLE
add information on (prospective) installation/deployment

### ISSUE THREAD

**Issue by bertsky** [OCR-D/ocr-d.github.io#6]
Please document how OCR-D workflows can be (packaged and) installed as a whole, including:

* Docker-based installation (configure composition of processors' docker files, or multi-stage build – automatically from given workflow configuration?)
* Docker-independent installation (gather all processors' requirements, run makefiles etc)

Granted, this might depend heavily on the workflow engine and how a workflow configuration will be described. But we could start with something that already works for `ocrd process`!

Probably the best place would be the guide?

**Comment by kba** [comment]
http://kba.cloud/2019-03-25-dhd/setup-time could be a starting pointy.

**Comment by bertsky** [comment]
Yes, along those lines. This looks like a lot of it can be scripted? Maybe we could create a repo `ocrd_install` which encapsulates all the docker / deb / pip installation steps? Probably even taking the desired workflow (or at least a list of needed processors) as input?
